### ISSUE TITLE
Screenshot is not working 

### ISSUE THREAD

**Issue by RR-Reddy** [rydmike/flex_scaffold_exp#2]
i am trying to take screenshot. and facing below issue.

![Screenshot 2024-02-28 at 10 35 11 AM](https://github.com/rydmike/flex_scaffold_exp/assets/32235964/034312ea-1b77-4b35-b795-81422e64bdda)

**Comment by rydmike** [comment]
Yes I know, this is due to issues in the used `device_preview` package back from when this demo was built in **2020**.

The `device_preview`  package is nice today, but it has also not been maintained in almost 2 years. So to be able to use it in the demo, I will probably need to fork it and fix the issues that bugs me in it.

There are already at least two community forks of `device_preview` on pub due to its lack of maintenance. Maybe I can get some of my fixes into the latest for. As for the screen shot feature of `device_preview` I will probably just disable it in `device_preview`. I don't know if it works OK on latest version, but I will test it of course.

The device preview is kind of nice in this responsive app demo since you can get a bit of feel of what things look like on different devices without actually building it on simulators/emulators or real devices.
